frontend/src/components/widgets/CameraInput/WebcamComponent.tsx

```diff
@@ -88,6 +95,16 @@ const WebcamComponent = ({
   )
   const videoRef = useRef<Webcam>(null)
 
+  const [debouncedWidth, setDebouncedWidth] = useState(width)
+  const memoizedSetDebouncedCallback = useCallback(
+    debounce(1000, setDebouncedWidth),
+    [debouncedWidth]
```

Comment:
I think we want to pass an empty dependency array to `useCallback` here since `debouncedWidth` changing doesn't have an effect on the function passed as the first parameter to `useCallback`

Currently, a new callback will be rendered every time the debounced function runs and . This doesn't change any behavior but does do more work than is required

Reply:
Done!